inc/saiport.h

```diff
@@ -4104,6 +4104,30 @@ typedef enum _sai_port_serdes_attr_t
      */
     SAI_PORT_SERDES_ATTR_RX_PRECODING,
 
+     /**
+     * @brief A collection of custom serdes attributes represented as a string
+     *
+     * The value is a string that includes a JSON object with key-value pairs.
+     * Each pair's key represents the serdes attribute name, and each pair's
+     * value is a list of values for every serdes lane. This allows
+     * vendor-specific serdes attributes to be forwarded in a JSON string
+     * without the sender needing to know the details. The sender simply passes
+     * along the data, vendor-defined rules determine which key-value pairs to
+     * include, and the vendor SDK interprets the JSON accordingly.
+     *
+     * Example of the JSON object:
+     *  {
+     *     "attr1": ["lane1_value", "lane2_value", "lane3_value", "lane4_value"],
+     *     "attr2": ["lane1_value", "lane2_value", "lane3_value", "lane4_value"],
+     *      ...
+     *  }
+     *
+     * @type sai_u8_list_t
```

Comment:
Should we use sai_json_t here instead?
https://github.com/opencomputeproject/SAI/blob/master/inc/saitypes.h#L1134-L1160

Reply:
If we use `sai_json_t`, does it enforce the JSON string to follow the format below (including metadata validation), or are we free to define any format for its contents?
https://github.com/opencomputeproject/SAI/blob/46e839113f19afff070c9ce5c277324bcbba398a/inc/saitypes.h#L1135-L1148

Additionally, here, the `"value"` should be of type `sai_s32_list_t`. Should we then represent it as JSON array, something like `"value": [10, 11, 12, 13]`?

Reply:
i think attribute format should be vendor specific

Reply:
Updated type to `sai_json_t`.

And according to [HLD](https://github.com/opencomputeproject/SAI/blob/master/doc/SAI_Proposal_Generic_Extensions.md#sai_json_t), `grammar` of this JSON is predefined, so I followed the same `grammar` to provide the example.
https://github.com/opencomputeproject/SAI/blob/46e839113f19afff070c9ce5c277324bcbba398a/doc/SAI_Proposal_Generic_Extensions.md?plain=1#L52